amplify.yml

```diff
@@ -7,6 +7,7 @@ frontend:
     build:
       commands:
         - rm -rf .next/
+        - cp .env.$USER_BRANCH .env.local
```

Comment:
important. This overrides the `.env.local` file so it uses the config for the branch we deploy